gh-127303: Add docs for token.EXACT_TOKEN_TYPES (#127304)
python/cpython: Doc/library/token.rst

   ``type_comments=True``.

.. data:: EXACT_TOKEN_TYPES

   A dictionary mapping the string representation of a token to its numeric code.

   .. versionadded:: 3.8

> 

Reviewer: Can we have two blank lines here? (as above)

   ``type_comments=True``.

.. data:: EXACT_TOKEN_TYPES

>    A dictionary that stores the string representation of the token
>    as keys, and their numeric codes as values.

Reviewer 2: How about this?
```suggestion
   A dictionary mapping the string representation of a token to its numeric code.
```

Reviewer 3: done
